lib/noop.js

```diff
@@ -0,0 +1 @@
+module.exports = require('lodash/utility/noop')
```

Comment:
I'm not, I wouldn't put it past jdd to have some sort of crazy `new Function` based black magic there